recipe-server/client/control_new/components/data/QueryExtensions.js

```diff
@@ -3,27 +3,38 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
-import { fetchAllExtensions } from 'control_new/state/extensions/actions';
+import * as extensionActions from 'control_new/state/extensions/actions';
```

Comment:
yeah it probably requires a larger discussion. and since it doesn't solve anything right now, i think we can defer. but it probably makes sense to file an issue about upgrading webpack and addressing this then.